src/modules/app/createReduxStore.js

```diff
@@ -20,12 +20,11 @@ export type ActionType<T> = { type: string, payload: Payload<T> }
 
 // todo: Change type to correct State type,
 // https://blog.callstack.io/type-checking-react-and-redux-thunk-with-flow-part-2-206ce5f6e705
-const createReduxStore = (createHistory: () => History, initialState: {} = {}, routesMap: RoutesMap = defaultRoutesMap): Store<any, any> => {
-  const history = createHistory()
-
-  const {reducer, middleware, enhancer} = connectRoutes(history, routesMap,
-    {onBeforeChange: onBeforeChange, querySerializer: queryString}
-  )
+const createReduxStore = (initialState: {} = {}, routesMap: RoutesMap = defaultRoutesMap): Store<any, any> => {
+  const { reducer, middleware, enhancer } = connectRoutes(routesMap, {
```

Comment:
redux-first-router now doesn't take a `history` as param anymore, but has one internally.
But scroll-behavior is not really good with the internal one: When you're already scrolled down and click on a link which leads to a long page, you already start "scrolled down" on the new page.

Reply:
Probably just not upgrade the router?

Reply:
Hm actually we implemented the scrolling behaviour. Thats nothing what the previous version did.

You can still use our custom version of history: https://github.com/faceyspacey/redux-first-router/blob/master/docs/migration.md

but maybe redux first router can also do the scrolling to the top.

Reply:
Here you go: https://github.com/faceyspacey/redux-first-router/blob/master/docs/connectRoutes.md#options

See scrollTop.